### PR TITLE
[zh-cn]sync dynamic-resource-allocation.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 1]
```diff
@@ -433,34 +345,180 @@ the `.spec.nodeName` field and to use a node selector instead.
...
+-->
+驱动程序可以报告每个已分配设备的驱动程序特定设备状态数据在资源声明中。例如，分配给网络接口设备的 IP
+可以在 ResourceClaim 状态中报告。
+
+驱动程序设置状态，信息的准确性取决于那些 DRA 驱动程序的实现。因此，报告的设备状态可能并不总是反映设备状态的实时变化。
+
+当此特性被禁用时，该字段会在存储 ResourceClaim 时自动清除。
+
+ResourceClaim 设备状态在可能的情况下是支持的，即从 DRA 驱动程序可以更新一个现有的 ResourceClaim，
+其中 `status.devices` 字段已被设置。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:492]:
```suggestion
针对一个已经设置了 `status.devices` 字段的现有 ResourceClaim 而言，如果 DRA
驱动能够更新该 ResourceClaim，则有可能支持 ResourceClaim 设备状态这一特性。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 2]
```diff
@@ -516,14 +565,38 @@ be installed. Please refer to the driver's documentation for details.
...
 欲了解详细信息，请参阅驱动程序的文档。
 
+<!--
+### Enabling admin access
+
+[Admin access](#admin-access) is an *alpha feature* and only enabled when the
+`DRAAdminAccess` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled in the kube-apiserver and kube-scheduler.
+-->
+### 启用管理员访问
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:575]:
```suggestion
### 启用管理性质访问  {#enabling-admin-access}
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 3]
```diff
@@ -516,14 +565,38 @@ be installed. Please refer to the driver's documentation for details.
...
+
+[管理员访问](#admin-access) 是一个 **Alpha 级别特性**，仅在 kube-apiserver 和 kube-scheduler
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:577]:
```suggestion
[管理性质访问](#admin-access) 是一个 **Alpha 级别特性**，仅在 kube-apiserver 和 kube-scheduler
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 4]
```diff
@@ -516,14 +565,38 @@ be installed. Please refer to the driver's documentation for details.
...
+
+<!--
+### Enabling Device Status
+
+[ResourceClaim Device Status](#resourceclaim-device-status) is an *alpha feature* 
+and only enabled when the `DRAResourceClaimDeviceStatus` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+is enabled in the kube-apiserver.
+-->
+### 启用设备状态
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:588]:
```suggestion
### 启用设备状态  {#enabling-device-status}
```

**Review by mengjiao-liu — COMMENTED** [review]
/approve

**Pull request merged**